### PR TITLE
Fix lint/format

### PR DESCRIPTION
**Pull request by bcomnes**
Ok so this:

- Disables prettier-eslint integration.  Make sure your editor is set to not take into account `eslint`!
- Use recommended rule set, with a few minor tweaks for preact.

### CODE REVIEW

File: src/components/forms/siteurl.js [hunk 1]
```diff
@@ -34,9 +34,11 @@ export default class SiteURLForm extends Component {
               name="url"
               value={url}
               placeholder="URL of your Netlify site"
-              autocapitalize="off"
+              autocapi
```
Comment (brycekahle) [inline src/components/forms/siteurl.js]: Whoa, this doesn't seem right.

Comment (bcomnes) [reply src/components/forms/siteurl.js]: oh whattt happened here?

Comment (bcomnes) [reply src/components/forms/siteurl.js]: I must have done that on accident.  Redoing the format completely didn't do that.

**Comment by brycekahle** [comment]
💯

**Pull request merged**